packages/features/ee/teams/lib/payments.ts

```diff
@@ -159,25 +217,8 @@ export const purchaseTeamOrOrgSubscription = async (input: {
 
     const priceObj = await stripe.prices.retrieve(fixedPriceId);
     if (!priceObj) throw new Error(`No price found for ID ${fixedPriceId}`);
-    try {
-      const customPriceObj = await stripe.prices.create({
-        nickname: `Custom price for ${isOrg ? "Organization" : "Team"} ID: ${teamId}`,
-        unit_amount: pricePerSeat * 100, // Stripe expects the amount in cents
-        // Use the same currency as in the fixed price to avoid hardcoding it.
-        currency: priceObj.currency,
-        recurring: { interval: "month" }, // Define your subscription interval
-        product: typeof priceObj.product === "string" ? priceObj.product : priceObj.product.id,
-        tax_behavior: "exclusive",
-      });
-      return customPriceObj.id;
-    } catch (e) {
-      log.error(
-        `Error creating custom price for ${isOrg ? "Organization" : "Team"} ID: ${teamId}`,
-        safeStringify(e)
-      );
 
-      throw new Error("Error in creation of custom price");
-    }
+    return priceObj;
```

Comment:
We don't need `pricePerSeat` variable in `getFixedPrice`, it is only required to create custom Price.

Reply:
I think we do as we need to get the priceId from stripe as we need the product and currency in createPrice